src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Razor/ExtractToCodeBehindCodeActionResolver.cs

```diff
@@ -104,17 +104,19 @@ public override async Task<WorkspaceEdit> ResolveAsync(JObject data, Cancellatio
 
             var start = codeDocument.Source.Lines.GetLocation(actionParams.RemoveStart);
             var end = codeDocument.Source.Lines.GetLocation(actionParams.RemoveEnd);
-            var removeRange = new Range(
-                new Position(start.LineIndex, start.CharacterIndex),
-                new Position(end.LineIndex, end.CharacterIndex));
+            var removeRange = new Range
+            {
+                Start = new Position(start.LineIndex, start.CharacterIndex),
+                End = new Position(end.LineIndex, end.CharacterIndex)
+            };
 
             var codeDocumentIdentifier = new OptionalVersionedTextDocumentIdentifier { Uri = actionParams.Uri };
             var codeBehindDocumentIdentifier = new OptionalVersionedTextDocumentIdentifier { Uri = codeBehindUri };
 
-            var documentChanges = new List<WorkspaceEditDocumentChange>
+            var documentChanges = new List<SumType<TextDocumentEdit, CreateFile, RenameFile, DeleteFile>>
```

Comment:
No need to use a List here, which means the `.ToArray()` can be removed in the return
```suggestion
            var documentChanges = new SumType<TextDocumentEdit, CreateFile, RenameFile, DeleteFile>[]
```